src/lang_utils/error_codes/M0198.md

```diff
@@ -0,0 +1,54 @@
+# M0198
+
+This warning means that you specified a field identifier in an object pattern without referencing this identifier later, a good indicator of dead code.
+
+Dubious code example:
+
+```motoko
+import Debug "mo:base/Debug";
```

Comment:
Question: I am not sure whether this code is shown in the web-based Motoko interpreter and whether it supports base library imports.

Reply:
I think VS code will display it, but not sure about the interpreter. I wouldn't worry about it now though.